charts/ingress-nginx/Chart.yaml

```diff
@@ -21,4 +21,5 @@ annotations:
   # List of changes for the release in artifacthub.io
   # https://artifacthub.io/packages/helm/ingress-nginx/ingress-nginx?modal=changelog
   artifacthub.io/changes: |
-    - Add namespace field in the namespace scoped resource templates
+    - Migrate the webhook-certgen program inside ingress repo.
+    - Fix forwarding of auth-response-headers to gRPC backends
```

Comment:
I had assumed that the entire list of changes has to be put here just like in the /Changelog.md

Reply:
Here is to follow the previous habit, only the most important changes are written

Reply:
ok, got it. thnx.